elimar/modules/users/fixtures/users_accounts.json

```diff
@@ -0,0 +1,502 @@
+[
+    {
+      "model": "users.user",
+      "pk": 1,
+      "fields": {
+        "first_name": "John_1",
+        "last_name": "Doe_1",
+        "email": "john@doe_1.com",
+        "password": "test"
```

Comment:
Shouldn't the password be hashed?

Reply:
I added those dummy users for tests/work crud actions on users.

Reply:
That why I did not hash passwords.